source/PPApp.cpp

```diff
@@ -11,6 +11,7 @@ void PanicPainterApp::onStartup() {
     _assets->attach<Texture>(TextureLoader::alloc()->getHook());
     _assets->attach<Sound>(SoundLoader::alloc()->getHook());
     _assets->attach<SceneNode>(Scene2Loader::alloc()->getHook());
+    _assets->attach<WidgetValue>(WidgetLoader::alloc()->getHook());
```

Comment:
@foxhatleo this took me so much time to figure out why widgets weren't working... Is there a reason we removed this line when we copied App.cpp initially?